olsrd/Makefile

```diff
@@ -298,6 +304,11 @@ define Package/olsrd-mod-watchdog/install
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/lib/watchdog/olsrd_watchdog.so.* $(1)/usr/lib/
 endef
 
+define Package/olsrd-utils/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DATA) ./files/olsrd-neigh.sh $(1)/usr/bin/olsrd-neigh.sh
```

Comment:
You have to istall that as binary. INSTALL_BIN. Maybe also just rename that in /usr/bin/olsrd-neigh

Reply:
I think it is better to use the name we are all familiar with neigh.sh